wolfcrypt/src/wolfmath.c

```diff
@@ -89,6 +89,30 @@ mp_digit get_digit(mp_int* a, int n)
     return (n >= a->used || n < 0) ? 0 : a->dp[n];
 }
 
+int mp_cond_copy(mp_int* a, int copy, mp_int* b)
```

Comment:
Looks good but please add a basic comment here describing why and what this does for ease of future maintenance.

Reply:
Comments added.